app/validators/paginationValidator.js

```diff
@@ -0,0 +1,16 @@
+const { query } = require('express-validator');
+
+async function isInt(value) {
```

Comment:
does returning true/false work?
`custom(page => !isNaN(page)).withMessage('Should be integer!')`?